android/app/src/main/java/io/mosip/registration_client/MainActivity.java

```diff
@@ -147,94 +153,131 @@ public class MainActivity extends FlutterActivity {
-            Log.e(getClass().getSimpleName(), "Registration : "+ registrationList);
-
-            registrationList = packetService.getRegistrationsByStatus("CREATED");
-            registrationList.forEach(value->{
+            Log.d(getClass().getSimpleName(), "Sync Packets in main activity");
+            Integer batchSize = getBatchSize();
+            List<Registration> registrationList = packetService.getRegistrationsByStatus(PacketClientStatus.APPROVED.name(), batchSize);
+            for (Registration value : registrationList) {
                 try {
-                    Log.d(getClass().getSimpleName(), "Syncing " + value.getPacketId());
+                    Log.e(getClass().getSimpleName(), "Syncing " + value.getPacketId());
                     packetService.syncRegistration(value.getPacketId());
-                }catch (Exception e){
+                } catch (Exception e) {
                     Log.e(getClass().getSimpleName(), e.getMessage());
                 }
-            });
+            }
+        }
+    }
 
-            registrationList = packetService.getRegistrationsByStatus("SYNCED");
-            registrationList.forEach(value->{
+    private void uploadRegistrationPackets(Context context) {
+        if(NetworkUtils.isNetworkConnected(context)){
+            Log.d(getClass().getSimpleName(), "Upload Packets in main activity");
+            Integer batchSize = getBatchSize();
+            List<Registration>  registrationList = packetService.getRegistrationsByStatus(PacketClientStatus.SYNCED.name(), batchSize);
+            for (Registration value : registrationList) {
                 try {
-                    Log.d(getClass().getSimpleName(), "Uploading " + value.getPacketId());
+                    Log.e(getClass().getSimpleName(), "Uploading " + value.getPacketId());
                     packetService.uploadRegistration(value.getPacketId());
-                }catch (Exception e){
+                } catch (Exception e) {
                     Log.e(getClass().getSimpleName(), e.getMessage());
                 }
-            });
+            }
+        }
+    }
+
+    private Integer getBatchSize(){
+        List<GlobalParam> globalParams = globalParamDao.getGlobalParams();
+        for (GlobalParam value : globalParams) {
+            if (Objects.equals(value.getId(), "mosip.registration.packet_upload_batch_size")) {
+                return Integer.parseInt(value.getValue());
+            }
         }
+        return 4;
+    }
+
+    private long getIntervalMillis(String api){
+        AtomicLong alarmTime = new AtomicLong(System.currentTimeMillis()+60000);
+        List<SyncJobDef> syncJobs = syncJobDefRepository.getAllSyncJobDefList();
+        for (SyncJobDef value : syncJobs) {
+            if (Objects.equals(value.getApiName(), api)) {
+                Log.e(getClass().getSimpleName(), String.valueOf(value.getSyncFreq()) + " Cron Expression");
```

Comment:
log.e should only be used where there's error being logged, use log.i or log.d for info or debug log

Reply:
Changed to log.d